python/tk_multi_launchapp/software_entity_launcher.py

```diff
@@ -228,10 +224,9 @@ def _get_sg_software_entities(self):
         # Now Group and User restrictions. Always retrieve Software entities
         # that have no Group or User restrictions.
         current_user = self._tk_app.context.user
-        user_group_filters = [
-            ["sg_user_restrictions", "is", None],
-            ["sg_group_restrictions", "is", None],
-        ]
+
+        # No user restriction filter.
+        user_group_filter = ["user_restrictions", "is", None]
```

Comment:
looks like we lost the group restrictions? if on purpose, update comment above and variable name, else should be added back in.